napari/util/misc.py

```diff
@@ -705,19 +723,28 @@ def get_keybindings_summary(keymap):
     keybindings_str : str
         String with summary of all keybindings and their functions.
     """
-    keybindings_str = ''
+    col = 'rgb(134, 146, 151)'
+    keybindings_str = '<table border="0" width="100%">'
     for key in keymap:
-        func_str = f'<b> {key}</b>: {get_function_summary(keymap[key])}<br>'
-        keybindings_str += func_str
-
+        keycodes = [KEY_SYMBOLS.get(k, k) for k in key.split('-')]
+        keycodes = "-".join(
+            [f"<span style='color: {col}'><b>{k}</b></span>" for k in keycodes]
+        )
+        keybindings_str += (
+            "<tr><td width='80' style='text-align: right; padding: 4px;'>"
+            f"<span style='color: rgb(66, 72, 80)'>{keycodes}</span></td>"
+            "<td style='text-align: left; padding: 4px; color: #CCC;'>"
+            f"{get_function_summary(keymap[key])}</td></tr>"
+        )
+    keybindings_str += '</table>'
     return keybindings_str
 
 
 def get_function_summary(func):
     """Get summary of doc string of function."""
     doc = FunctionDoc(func)
     summary = ''
-    summary += doc['Signature']
+    # summary += doc['Signature']
```

Comment:
why the comment?

Reply:
Hi @kne42!  Welcome back, I've heard so much about you and read so many of your commits :) Nice to meet you.  The comment was to remove the signature from the function summary as mentioned briefly [above](https://github.com/napari/napari/pull/749#issuecomment-560189455).  I know it's a bit of a change, but I felt like for the public facing window like this the signature was unnecessary (almost all of the signatures are duplicates of the docstring).  We could just delete the line fully, but i commented so as not to forget how it worked if we wanted to change our minds.

Reply:
Nice to meet you too! I'm so happy that you're passionate about this project too, and am excited to see you join the core development team :)

I'm not a big fan of commenting instead of deleting, but if you add another comment explaining why it's commented out, it should be fine :)

Reply:
@kne42 :wave::wave::wave::wave::wave: so good to see you back. :grin: 

@tlambert03 I'm with @kne42, I would say remove it, but it's true that uncommenting a line is less effort than researching how to do this in the first place. An alternative would be:

 ```suggestion
    # if we want to add the function signature to the summary, use:
    # summary += doc['Signature']
```

Reply:
I'm ok with dropping, isn't too hard to get the signature back. I also really like how much cleaner it is without